bus/bus.go

```diff
@@ -448,12 +447,12 @@ func (b *Bus) Handler() http.Handler {
 		"POST   /multipart/listuploads": b.multipartHandlerListUploadsPOST,
 		"POST   /multipart/listparts":   b.multipartHandlerListPartsPOST,
 
-		"GET    /objects/*path":  b.objectsHandlerGET,
-		"PUT    /objects/*path":  b.objectsHandlerPUT,
-		"DELETE /objects/*path":  b.objectsHandlerDELETE,
-		"POST   /objects/copy":   b.objectsCopyHandlerPOST,
-		"POST   /objects/rename": b.objectsRenameHandlerPOST,
-		"POST   /objects/list":   b.objectsListHandlerPOST,
+		"GET    /listobjects/*prefix": b.objectsHandlerGET,
```

Comment:
I'm wondering if we can also fit `/search/objects` into this endpoint. Theoretically if no delimiter is set we could check a `substring` parameter. Then it's essentially the same.

Reply:
Kind of quirky 🤷‍♂️ but yeah we definitely could.

Reply:
Yeah still thinking about it but it would be nice to get rid of `/search` cause it's very out-of-place.

Reply:
Yeah I agree, in hindsight though, it would've been less out of place if we had expanded it and we had a UI on top that would extensively use it for list views or something. Removing it probably better option though.